docs/_templates/navigation.html

```diff
@@ -10,23 +10,6 @@ <h3>Job Postings</h3>
         </ul>
 </div>
 
-<div id="toc">
-
-    <h3>Site Content</h3>
-
-    {{ toctree(collapse=False, includehidden=theme_sidebar_includehidden) }}
```

Comment:
I do think this will make it so some of our content isn't linked anywhere and can't be indexed by Google. I think we probably want to add a page that is a full TOC that is linked from somewhere?

Reply:
Good point, I've added that in e38f6920 and linked it from the footer